src/main/java/com/tstepnik/planner/repository/TaskRepository.java

```diff
@@ -15,4 +16,10 @@
 
     Optional<Task> findById(Long id);
 
+    @Query("SELECT COUNT(t) FROM Task t WHERE t.userId = ?1 AND t.plannedFor <= ?2")
```

Comment:
Why are you writing your own queries if you dont have to?

Reply:
Because method name is too long:
countAllUserArchivedTasks()  - method with query,
countAllByUserIdAndPlannedForBefore() - spring method without query.

Reply:
i could even remove word "All" and my method will be even shorter.

Reply:
Lol, I guess that's a good point, but wait until you end up in commercial project. You will see long method names there :D